package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "react-native-country-picker-modal",
-  "version": "2.0.0",
-  "description": "react-native country picker",
+  "version": "3.0.0",
+  "description": "react-native country picker - updated dependencies",
```

Comment:
why should the description change? That doesn't appear to make sense
PRs that bump versions are generally difficult for maintainers to handle, in my experience maintaining other packages.
I'd remove this

```suggestion
```